variables.tf

```
@@ -10,33 +11,22 @@ variable "stage" {
   description = "Stage (e.g. `prod`, `dev`, `staging`)"
 }
 
-variable "delimiter" {
-  type        = "string"
-  default     = "-"
-  description = "Delimiter to be used between `name`, `namespace`, `stage`, etc."
-}
-
 variable "attributes" {
   type        = "list"
   default     = []
   description = "Additional attributes (e.g. `policy` or `role`)"
 }
 
-variable "tags" {
-  type        = "map"
-  default     = {}
-  description = "Additional tags (e.g. `map('BusinessUnit','XYZ')`)"
-}
-
 variable "datadog_external_id" {
   description = "External Id of the DataDog service"
```

```suggestion
  description = "External Id of the Datadog service"
```

Use the canonical name